LiskHQ/lisk-hub
# Implement the details of the block on a dedicated page - closes #2505

### What issue have I solved?

-- #2505 

### How have I implemented/fixed it?

This PR include the page to see the details about any block in the network.

- [x] Add the new route to the router file.
- [x] Create the new component.
- [x] Implement the Block details table.

This PR is only to see the first table in the Block Details page NOT the transaction list.

### How has this been tested?

To test this you need to use Mainnet or Testnet, Devnet wont work.

1. Run the app.
2. Select Testnet network.
3. In the URL write <localhost_url>/monitor/blocks
4. Then you will see the list of all blocks.
5. Select any block by doing click in the block ID.
6. The Block Details page will show up.
7. Now you can see all the block details in this page. (ONLY block details NO transaction list).



### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
